### PR TITLE
[16.0][MIG] product_template_multi_link

### PR DESCRIPTION
**Pull request by sbejaoui**
based on #684

### CODE REVIEW

**Review by lmignon — APPROVED** [review]
LGTM, code review + functional test

File: product_template_multi_link/demo/product_template_link.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
```
Comment (hieulucky111) [inline product_template_multi_link/demo/product_template_link.xml:2]: should be `noupdate="1"`

File: product_template_multi_link/models/product_template.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    _inherit = "product.template"
+
+    product_template_link_ids = fields.One2many(
+        string="Product Links",
+        comodel_name="product.template.link",
+        compute="_compute_product_link_ids",
+    )
+
+    product_template_link_count = fields.Integer(
+        string="Product Links Count", compute="_compute_product_template_link_count"
```
Comment (hieulucky111) [inline product_template_multi_link/models/product_template.py:21]: i think we can use the same compute function `_compute_product_link_ids`

**Review by hieulucky111 — COMMENTED** [review]
Code Review.
LGTM, just minor change.

**Review by sebalix — APPROVED** [review]

**Comment by lmignon** [comment]
@OCA/e-commerce-maintainers can you merge this one PLZ.

**Pull request merged**